elastic/kibana
# [ObsUx][Inventory] Add actions column with link to discover for inventory

Closes #199025 
## Summary

This PR adds an actions column with a link to discover for inventory. It is available in the inventory grid(s) in both the group view and the unified inventory view. 

The column header tooltip text will change when it's available: [issue](https://github.com/elastic/kibana/issues/199500) added

⚠️ If the discover link is not available I added a logic to hide the whole actions column as this is the only available option for now. Once we add more actions we should refactor that to just not add the action and to keep the column visible (which doesn't make sense atm)

## Testing
- Enable the Inventory
- Check with/without grouping both the action link and the button
   - combination of kuery / drop-down filter
   - without any filters
   - With just one kuery or drop-down filter 
   - When the link is clicked from the table we should see a filter by identity field in the query in discover (like `service.name: 'test'`, `conteainer.id: 'test'`)
  

https://github.com/user-attachments/assets/bb4a89f5-2b30-457f-bf13-7580ff162a7e


https://github.com/user-attachments/assets/2894ef5c-6622-4488-ab84-c453f5b6e318
